script/run-container.sh

```diff
@@ -125,6 +125,11 @@ container_tnf_kubeconfig_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_
 CONTAINER_TNF_DOCKERCFG=$(join_paths "${container_tnf_dockercfg_paths[@]}")
 container_tnf_dockercfg_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_dockercfg_volume_bindings[@]}")
 
+# Safeguard against an empty variable
+if [ -z "$CONTAINER_TNF_DOCKERCFG" ]; then
+	CONTAINER_TNF_DOCKERCFG="NA"
```

Comment:
There is no need for double quotes, `CONTAINER_TNF_DOCKERCFG=NA` is fine.

Reply:
Done.